CIP-1694/README.md

```diff
@@ -0,0 +1,624 @@
+
+### The Cardano Constitution
+
+The Constitution is a text document that defines Cardano's shared values and guiding principles. At this stage, it is meant to be an informational document that unambiguously captures the Cardano core values. At a later stage, we can imagine the Constitution perhaps evolving into a smart-contract-based set of rules driving the entire governance framework. For now, however, the Constitution will remain an off-chain document whose hash digest value will be recorded on-chain.
+
+### The Constitutional Committee
+
+We define a _Constitutional Committee_ which represents a set of individuals or entities (associated with a pair of Ed25519 credentials) who are responsible for overseeing the governance actions that are defined in the section below and ensuring that the Constitution is respected.
+
+The Constitutional Committee is considered to be in one of the following two states at all times:
+
+1. a normal state (i.e. one of confidence); or
+2. a state of no-confidence
+
+In a _state of no-confidence_, the current committee is no longer able to participate in governance actions
+and must be replaced before any governance actions can be enacted (see below).  Any outstanding governance actions become void immediately the committee
+enters a state of no-confidence.
+
+The Constitutional Committee will use a hot and cold key setup. Hot keys will re-use the existing "genesis delegation certificate" mechanism that has been in place since the start of the Shelley era.
+
+#### Initial Constitutional Committee
+
+The initial Constitutional Committee will constitute the core members of a member-based organization that is dedicated to the development of Cardano. The final list of members is yet to be defined. However, it will, in all likelihood, be made of some of the founding entities, such as Input Output Global and the Cardano Foundation, as well as key community actors who are interested in participating in the Cardano governance process.
+
+#### Replacing the Constitutional Committee
+
+The Constitutional Committee can be replaced in one of two ways:
+
+* When in a **normal** state (i.e. a state of **confidence**), the committee can be replaced via a specific governance action (action 2 below) that requires the approval of both the **current Constitutional Committee** and the **DReps**.
+
+* When in a state of **no-confidence**, the committee can also be replaced via a specific governance action (action 5 below), but this requires the approval of the **SPOs** and the **DReps** instead.
+
+#### Size of the Constitutional Committee
+
+Unlike the Shelley governance design, the size of the Constitutional Committee is not fixed.  It may be changed any time that a new committee is installed.  Likewise, the _quorum_ (the number of votes that are required to enact governance actions) is not fixed and can be varied whenever a new committee is installed. This gives a great deal pf flexibility.
```

Comment:
```suggestion
Unlike the Shelley governance design, the size of the Constitutional Committee is not fixed.  It may be changed any time that a new committee is installed.  Likewise, the _quorum_ (the number of votes that are required to enact governance actions) is not fixed and can be varied whenever a new committee is installed. This gives a great deal of flexibility.
```